### PR TITLE
added retry-after supported resources for 429

### PR DESCRIPTION
**Pull request by baywet**
retry-after header is not supported by all resources due to implementation of the underlaying API's
this PR documents which resources are supported at the moment
https://stackoverflow.com/questions/47146598/not-receiving-retry-after-headers-from-ms-graph-api
https://blogs.msdn.microsoft.com/exchangedev/2017/04/07/throttling-coming-to-outlook-api-and-microsoft-graph/
https://officespdev.uservoice.com/forums/224641-feature-requests-and-feedback/suggestions/32797489-consistent-retry-after-information-accross-differe
https://github.com/microsoftgraph/msgraph-sdk-javascript/issues/42

### CODE REVIEW

File: concepts/throttling.md [hunk 1]
```diff
@@ -30,4 +30,15 @@ When you implement error handling, use the HTTP error code 429 to detect throttl
 2. Retry the request.
 3. If the request fails again with a 429 error code, you are still being throttled. Continue to use the recommended Retry-After delay and retry the request until it succeeds.
 
+The following resources currently provide a retry-after header:
+- [User resources](https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/resources/user)
+- [Photo resources](https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/resources/profilephoto)
```
Comment (Lauragra) [inline concepts/throttling.md]: Please change all the links to relative, rather than absolute, links.

File: concepts/throttling.md [hunk 2]
```diff
@@ -30,4 +30,15 @@ When you implement error handling, use the HTTP error code 429 to detect throttl
 2. Retry the request.
 3. If the request fails again with a 429 error code, you are still being throttled. Continue to use the recommended Retry-After delay and retry the request until it succeeds.
 
+The following resources currently provide a retry-after header:
+- [User resources](https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/resources/user)
```
Comment (Lauragra) [inline concepts/throttling.md]: To reduce repetition, can we remove the word "resources" from the link text? Note that we refer to resources in the introductory sentence, so the context is clear.

File: concepts/throttling.md (hunk 1 above)
Comment (baywet) [reply concepts/throttling.md]: At what path segment do I need to cut it?

File: concepts/throttling.md (hunk 2 above)
Comment (baywet) [reply concepts/throttling.md]: done

File: concepts/throttling.md (hunk 1 above)
Comment (Lauragra) [reply concepts/throttling.md]: Just put the "../" right before the "api-reference/" folder. Thanks!

Comment (baywet) [reply concepts/throttling.md]: done

File: concepts/throttling.md [hunk 3]
```diff
@@ -30,4 +30,15 @@ When you implement error handling, use the HTTP error code 429 to detect throttl
...
+The following resources currently provide a retry-after header:
+- [User](../api-reference/v1.0/resources/user)
+- [Photo](../api-reference/v1.0/resources/profilephoto)
+- [Mail](../api-reference/v1.0/resources/message)
+- [Calendar (users and groups)](../api-reference/v1.0/resources/event)
+- [Contact](../api-reference/v1.0/resources/contact)
+- [Attachment](../api-reference/v1.0/resources/attachment)
+- [Group conversations](../api-reference/v1.0/resources/conversation)
+- [People and social](../api-reference/beta/resources/social_overview)
+- [Drive resources (OneDrive)](../api-reference/v1.0/resources/drive)
```
Comment (Lauragra) [inline concepts/throttling.md]: One more to delete.

File: concepts/throttling.md [hunk 4]
```diff
@@ -30,4 +30,15 @@ When you implement error handling, use the HTTP error code 429 to detect throttl
 2. Retry the request.
 3. If the request fails again with a 429 error code, you are still being throttled. Continue to use the recommended Retry-After delay and retry the request until it succeeds.
 
+The following resources currently provide a retry-after header:
+- [User](../api-reference/v1.0/resources/user)
```
Comment (Lauragra) [inline concepts/throttling.md]: Sorry, we also need the ".md" extension after the filename in the links. Thanks!

Comment (baywet) [reply concepts/throttling.md]: no worries, updated

File: concepts/throttling.md (hunk 3 above)
Comment (baywet) [reply concepts/throttling.md]: sorry about that, doing things a bit too fast, updated.

**Pull request merged**